src/OrleansSQLUtils/Properties/AssemblyInfo.cs

```diff
@@ -21,4 +21,5 @@
 [assembly: InternalsVisibleTo("UnitTests")]
 [assembly: InternalsVisibleTo("TesterInternal")]
 [assembly: InternalsVisibleTo("UnitTestGrains")]
+[assembly: InternalsVisibleTo("Orleans.NonSiloTests")]
```

Comment:
it is for `GarewaySelection_SqlServer` test, in `GatewaySelectionTests`. It uses a `SqlMembershipTable `as a `GatewayListProvider`. Since this is a Gateway selection tests, not a provider specific test, I wasn't sure where to put it. 

Do you still think we should keep it in original project for now?